MeshLib/Elements/LineRule2.h

```diff
@@ -39,6 +39,9 @@ class LineRule2 : public EdgeRule
     /// Constant: The FEM type of the element
     static const CellType cell_type = CellType::LINE2;
 
+    /// Constant: The number of edges
+    static const unsigned n_edges = 1;
```

Comment:
It seems to be the only occurence in this whole PR, so probably it can be savely removed?